File: ATP/1_termo_2_bimestre_ATP/Exercicio_Extra_de_Vetores/ex2.c
#include <stdio.h>
#define TF 5
int main(){
    int numVoo[TF], lugDisp[TF], idem, voo, res
    for(int i=0;i<TF;i++){
        printf("Digite o numVoo:\n");
        scanf("%d", &numVoo[i]);
    }
    for(int i=0;i<TF;i++){
        printf("Digite o lugares disponiveis do numVoo %d\n", numVoo[i]);
        scanf("%d", &lugDisp[i]);
    }
    printf("Digite o identidade:\n");
    scanf("%d", &idem);
    while(idem!=9999){
        printf("Digite o numero de voo:\n");
        scanf("%d", &voo);
        for(int i=0;i<TF;i++){
            if(numVoo[i]==voo){
                lugDisp[i]--;
                res= lugDisp[i];
                goto label;
            }
            else{
                printf("numero invalido\n");
            }
        }
        label:
        printf("numero de vagas:%d\n", res);
        printf("Digite o identidade:\n");
        scanf("%d", &idem);
    }
}

File: 1 Termo/AP/02_05_2024_Python_Estruturas_de_Repetição_Exercícios/Ex2.py
b=0
l=1
maior=0
menor=0
b=int(input("digiteo o numero:"))
while b!=0:
    if b<0:
        print("numero invalido")
    if(l==1):
        maior = b;
        menor = b;
    l=0
    if maior < b and b>0:
        maior = b;
    if menor > b and b>0:
        menor = b;
    b = int(input("digite o numero:"))
print(f"maior numero eh {maior}")
print(f"menor numero eh {menor}")
